src/test/resources/org/collectionspace/qa/cucumber/general_pages/front_page_and_login.feature

```diff
@@ -10,10 +10,10 @@ Feature: #Enter feature name here
     Then a drop down list should appear in the "Person" row
     Then the list in the "Person" row should contain "Local Persons, ULAN Persons"
       And clicks on the Create button
-    Then the vocabulary name in the titlebar should contain "Local Person"
+    Then the titlebar should contain "Local Person"
```

Comment:
I seem to have inadvertently altered this file. Is there a way to reject this change, or should I check it back out from collectionspace/master and re-commit?